src/projectscene/view/tracksitemsview/selectioncontextmenumodel.cpp

```
@@ -21,12 +21,48 @@ void SelectionContextMenuModel::load()
+    MenuItemList cutAndItems {
+        makeMenuItem("cut-leave-gap",
+                     muse::TranslatableString("selection", "Cut and leave gap")),
+        makeMenuItem("cut-per-clip-ripple",
+                     muse::TranslatableString("selection", "Cut and close gap")),
+        makeMenuItem("cut-per-track-ripple",
+                     muse::TranslatableString("selection", "Cut and close gap on this track")),
+        makeMenuItem("cut-all-tracks-ripple",
+                     muse::TranslatableString("selection", "Cut and close gap on all tracks")),
+    };
+
+    MenuItemList pasteAndItems {
```

Cppcheck (2.20.0)

[style] The function 'fileName' is never used.

(unusedFunction)